### ISSUE TITLE
OS build number incomplete

### ISSUE THREAD

**Issue by steffmeister** [OCSInventory-NG/WindowsAgent#158]
### General information
Operating system :  Windows 10

### OCS Inventory information
Windows agent version : 2.4

### Problem's description
OCS agent does not collect the full os build version number? eg for my computer (Win10-1909) it says, 10.0.18363, but I would expect 18363.657, so either we add the .657 to os build or we add another field named "os build" (or something like that).
I can write a patch either way (at least I think so), but someone from the core team should say how this should be handled, extend os version or add a seperate field.

**Comment by Tarbos** [comment]
This is the value **UBR** (Update Build Revision) from the key **SOFTWARE\Microsoft\Windows NT\CurrentVersion**

**Comment by gillesdubois** [comment]
Hi @steffmeister, 

Please try our OSInstall plugin that will retrieve more detailled informations on OS build and options.

Regards,
Gilles Dubois.

**Comment by pchamo** [comment]
I tried the OSInstall plugin version 2, but it doesn't get the osbuild value. I opted to add a registry data (osversion-osbuild) for HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\UBR (The plugin gets values like this 17763.1.x86fre.rs5_release.180914-1434 and not 1999 as shown by winver (like this OS Build 17763.1999)). And I added a new field to query the osbuild from all computers page (fields_118 in my case), and added this sql line to get those values filled:

mysql -D ocsweb -s -e "UPDATE accountinfo a, (SELECT hardware_id,name,regvalue from registry)r SET a.fields_118 = r.regvalue WHERE r.name='osversion-osbuild' AND r.regvalue IS NOT NULL and a.hardware_id=r.hardware_id".

Thanks.
